react-carousel/src/tools/simulateEvent.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Simulates mouse events when touch events occur
+ * @param {event} e A touch event
+ */
+const simulateEvent = (e) => {
```

Comment:
could you rename `e` to `event`?

and in other files

Reply:
ok, but I think everyone who works on the frontend knows that the first param in event callbacks always refer to event